README.md

```diff
@@ -16,9 +16,11 @@ The St. Olaf community, now in pocket size… rewritten in React Native.
 
 - [Install React Native](http://facebook.github.io/react-native/docs/getting-started.html#content)
 - [Install Yarn](https://yarnpkg.com/en/docs/install)
+- [Git Large File Storage (LFS)](https://help.github.com/articles/installing-git-large-file-storage/)
 - Clone the repository
 - `cd` into your clone
 - Run `yarn`
+- Run `git lfs pull`
```

Comment:
I still want `git lfs install` before this. However I found my original instructions, I managed to not get it activated on this repo. It's an idempotent operation, more or less, though, so I'd still like to put it here. Please?